src/pages/ProductView/DetailProduct.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+const DetailProduct = ({ viewData }) => {
+  return (
+    <div className="detailProduct">
+      <div id="detail">
+        <div className="tabBox">
+          <a className="tab clicked" href="#detail">
+            상품상세정보
+          </a>
+          <a className="tab unclicked" href="#delivery">
+            배송/교환 및 반품안내
+          </a>
+        </div>
+
+        <div className="detailContent">
+          <img
+            className="contentImage"
+            src="images/detail1.png"
+            alt="details"
+          />
+          <div className="detailComponents">
+            <div className="components">
+              <div className="compoTitle">제품 성분</div>
+              <br />
+              <li className="compoSubTitle">원재료의 특성에 한함</li>
+              대표 성분
+              <div>
+                {viewData &&
+                  viewData.components?.map((val, idx) => {
+                    if (val.important === true && idx === 0) {
+                      return idx === 0 ? (
+                        <span key={idx}>{val.name}</span>
+                      ) : (
+                        <span key={idx}>, {val.name}</span>
+                      );
```

Comment:
::after / ::before 가상 선택자와 content 속성을 활용하면 
```suggestion
                    return val.important && (<span key={idx}>{val.name}</span>)
```
이렇게만 해도 될 것 같네요! 참고해주세요

Reply:
그리고 key에 idx를 사용하는 것은 불가피한 경우가 아니면 지양해주시고 가급적이면 데이터의 id를 활용해주세요!